templates/virus_scan.sh

```diff
@@ -1,18 +1,28 @@
 #!/bin/bash
+# Managed by ansible role clamscan
 
 set -o nounset
 set -o pipefail
 
 LAST_SCAN_LOG_FILENAME='/var/log/clamav/lastscan.log'
 LAST_DETECTION_FILENAME='/var/log/clamav/last_detection'
 
-# Scan the entire file system (modulo the /dev, /sys, and /proc trees)
+# Scan the entire file system (modulo excluded trees)
 # and write to the log
 clamscan --infected --recursive \
```

Comment:
This command should be updated so that all of the clamscan flags are listed alphabetically.

Reply:
To me it just make more sense to have the options specifically handed by the role at the beginning, and the vague set of additional and facultative options at the end. Hence the order both in the script and in the defaults. No big deal though.